src/moscot/backends/ott/_solver.py

```diff
@@ -268,18 +268,40 @@ def _prepare(
         xy: Optional[TaggedArray] = None,
         x: Optional[TaggedArray] = None,
         y: Optional[TaggedArray] = None,
+        alpha: float = 0.5,
         **kwargs: Any,
     ) -> QuadraticProblem:
-        del xy
         if x is None or y is None:
             raise ValueError(f"Unable to create geometry from `x={x}`, `y={y}`.")
-        geom_x = self._create_geometry(x, **kwargs)
-        geom_y = self._create_geometry(y, **kwargs)
+        geom_xx = self._create_geometry(x, **kwargs)
+        geom_yy = self._create_geometry(y, **kwargs)
+        if alpha == 0 or xy is None:
+            # arbitrary positive fused penalty
+            geom_xy, fused_penalty = None, 1.0
```

Comment:
can we set `fused_penalty` to 0.0 then? this makes more sense I think.

Reply:
It has to be positive, since we check for this in `ott`; updated the comment.